src/remoteConnector.ts

```diff
@@ -504,7 +515,7 @@ export default class RemoteConnector extends Disposable {
 			this.logger.error(`Couldn't write '${sshDestInfo.hostName}' host to known_hosts file:`, e);
 		}
 
-		let identityFilePaths = await checkDefaultIdentityFiles();
+		let identityFilePaths = await this.getIdentityFilePaths(gitpodHost);
```

Comment:
I think it would be better to use `sshDestInfo.hostName` if there's a rule `*.gitpod.io` the ssh-config library will match it too, could you try that